Config/FilterFactoryCollectionInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the `liip/LiipImagineBundle` project.
+ *
+ * (c) https://github.com/liip/LiipImagineBundle/graphs/contributors
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Liip\ImagineBundle\Config;
+
+use Liip\ImagineBundle\Factory\Config\FilterFactoryInterface;
+
+interface FilterFactoryCollectionInterface
```

Comment:
imho there is only one way how this can be implemented and hence no point in having an interface

Reply:
done